browse-next/src/views/DeviceDiagnosticsSubView.vue

```diff
@@ -454,7 +767,123 @@ const deviceLocationPoints = computed<NamedPoint[]>(() => {
       </div>
     </div>
     <div class="mt-4" v-if="device.type === 'thermal'">
+      <h6 class="mt-4">Device Settings:</h6>
+      <span v-if="settingsLoading">
+        <b-spinner small class="me-2" />
+      </span>
+      <div v-else-if="settings" class="device-settings mt-3">
+        <div><b>Synced:</b> {{ settings.synced ? "Yes" : "No" }}</div>
+        <b-table
+          :items="settingsTable"
+          :fields="fields"
+          striped
+          hover
+          class="settings-table mt-3"
+        >
+          <template #cell(actions)="row">
+            <div v-if="row.item.name === 'Use Low Power Mode'">
+              <b-button
+                @click="handleToggleUseLowPowerMode(row.item)"
```

Comment:
Toggling low power mode should only be available on devices that support it.  One way to work out what kind of device we're looking at currently is to check the `saltNodeGroup` variable on this page for a substring of `tc2`.